src/main/java/net/imagej/ops/geom/geom3d/DefaultVoxelization3D.java

```diff
@@ -0,0 +1,394 @@
+package net.imagej.ops.geom.geom3d;
+
+import net.imagej.ops.OpService;
+import net.imagej.ops.Ops;
+import net.imagej.ops.geom.geom3d.mesh.DefaultMesh;
+import net.imagej.ops.geom.geom3d.mesh.Facet;
+import net.imagej.ops.geom.geom3d.mesh.Mesh;
+import net.imagej.ops.geom.geom3d.mesh.TriangularFacet;
+import net.imagej.ops.special.function.AbstractUnaryFunctionOp;
+import net.imglib2.FinalInterval;
+import net.imglib2.RandomAccess;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.RealLocalizable;
+import net.imglib2.RealPoint;
+import net.imglib2.img.Img;
+import net.imglib2.type.logic.BitType;
+import java.util.Set;
+
+import org.apache.commons.math3.geometry.euclidean.threed.Vector3D;
+import org.scijava.ItemIO;
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+
+
+/**
+ * This is a voxelizer that produces a binary image with values filled in along the surface of the mesh.
+ * 
+ * Thanks to Tomas Möller for sharing his public domain code:
+ * http://fileadmin.cs.lth.se/cs/personal/tomas_akenine-moller/code/tribox.txt
+ *
+ * @author Kyle Harrington, University of Idaho
```

Comment:
Please put your affiliation in parentheses, for consistency with others who are doing that.